src/app/features/notification/notification.service.ts

```diff
@@ -2,7 +2,6 @@ import { inject, Injectable } from "@angular/core";
 import { Logging } from "app/core/logging/logging.service";
 import { HttpClient } from "@angular/common/http";
 import { KeycloakAuthService } from "app/core/session/auth/keycloak/keycloak-auth.service";
-import { AngularFireMessaging } from "@angular/fire/compat/messaging";
```

Comment:
can you revert the change/reformating here, to keep the git changes as clean as possible? :blush:

Reply:
as you see here the changes compared with master, there is still a difference (the import, which seems to actually be used and required by the code in `master`, is now deleted). I suggested to get this file into the state that's exactly matching the master branch, so that we don't make accidental changes.